analyzers/tests/SonarAnalyzer.UnitTest/Common/ReadMeTest.cs

```diff
@@ -35,7 +35,7 @@ public class ReadMeTest
 
     [TestInitialize]
     public void Init() =>
-       readMe = File.ReadAllText(Path.Combine(TestContext.DeploymentDirectory, @".\..\..\..\..\..\..\README.md"));
+       readMe = File.ReadAllText(Path.Combine(TestContext.DeploymentDirectory, PathResolver.RelativePathToTestProjectRoot, @"..\..\..\README.md"));
```

Comment:
The altcover tool has a limitation. It has to be invoked without a parameter for the project/solution path. Due to this we have to call it from the analyzers folder and the working directory is changed when running in CI context.